core/integration/features/verified/0034-PROB-liquidity_TauScaling.feature

```diff
@@ -0,0 +1,604 @@
+Feature: Tests impact from change of tau.scaling parameter on probability of trading, 0038-OLIQ-006
+
+  Scenario: 001: set tau.scaling to 1
+  Background:
+    Given the following network parameters are set:
+      | name                                                | value |
+      | market.value.windowLength                           | 1h    |
+      | market.stake.target.timeWindow                      | 24h   |
+      | market.stake.target.scalingFactor                   | 1     |
+      | market.liquidity.targetstake.triggering.ratio       | 0     |
+      | market.liquidity.providers.fee.distributionTimeStep | 10m   |
+      | network.markPriceUpdateMaximumFrequency             | 0s    |
+      | market.liquidity.probabilityOfTrading.tau.scaling   | 10    |
```

Comment:
On line 3 we say tau scaling is 1, is 10 what we want here? Because then the next scenario also looks like the scaling is 10.